src/engine/math_util.c

```diff
@@ -53,14 +58,24 @@ u32 random_u16(void) {
     return gRandomSeed16;
 }
 
+// Generate a pseudorandom integer from -32768 to 32767.
+s16 random_s16(void) {
```

Comment:
This is both potentially unsafe and probably not very useful. Please remove this one.